mobify/nightwatch-commands
# Use selenium download

Status: **Ready for Review** 
Owner: 
Reviewers:  @marlowpayne @mikenikles

## Changes
- Latest version of Chrome (54) is too new for our outdated version of Chromedriver. This PR switches over to using https://github.com/groupon/selenium-download to manage the downloading of the latest versions of Selenium server and Chromedriver, with some moving/renaming to keep the same file structure as previous versions of nightwatch-commands

## Todos:
- [x] Passed linting check (run `grunt lint`)

### Feedback:
_none so far_

## How To Test
- Checkout this branch
- `npm install`
- `npm link`
- `grunt lint`
-  In `package.json`, change the nightwatch-commands line to `"nightwatch-commands": "mobify/nightwatch-commands#use-selenium-download",`, then run the end-to-end tests. You will need Chrome 54 on your machine to test locally.
- To test on CircleCI, update `circle.yml` to use node 4 and the latest stable version of Chrome. See  machine and dependencies https://github.com/mobify/lancome-progressive/blob/automation_construction/circle.yml
- Or just check out https://github.com/mobify/lancome-progressive/tree/automation_construction
